src/main/java/justin/command/DeleteCommand.java

```diff
@@ -0,0 +1,43 @@
+package justin.command;
+
+import justin.DukeException;
+import justin.Storage;
+import justin.task.Task;
+import justin.TaskList;
+import justin.Ui;
+
+/**
+ * Represents a command to delete a particular
+ * task from the TaskList.
+ * @author Justin Cheng.
+ */
+public class DeleteCommand extends Command {
+    private int num;
+
+    /**
+     * Constructor for the DeleteCommand class.
```

Comment:
I checked. Your command classes are documented really comprehensively 👍🏻 for overridden methods, prof said it's not necessary but up to us.